lib/Agrammon/DB/Dataset.pm6

```diff
@@ -85,31 +96,62 @@ class Agrammon::DB::Dataset does Agrammon::DB {
     has Bool $.read-only;
     has Str  $.model;
     has Str  $.comment;
-    has Str  $.version;
-    has Int  $.records;
+    has Str  $.version is default('2.0-stage'); # TODO: get from config
+    has Int  $.records; # TODO: is this needed?
     has DateTime $.mod-date;
     has $.data;
     has Agrammon::DB::Tag  @.tags;
     has Agrammon::DB::User $.user;
 
-    method create {
+    method create-dataset( $dataset-name, $username, $version, $model ) {
```

Comment:
If this is factored out for the use of `create` and `clone`, maybe it should be a private method (`method !create-dataset(...)` and the call it as `self!create-dataset(...)`).

Reply:
Done.